docs/source/learn/core_notebooks/Gaussian_Processes.rst

```diff
@@ -0,0 +1,267 @@
+:orphan:
+
+..
+    _href from docs/source/index.rst
```

Comment:
This should be removed. In their place, I would add a target so we can easily reference this page from other docs or notebooks from pymc-examples